cmd/otelcontribcol/Dockerfile

```diff
@@ -9,7 +9,8 @@ ARG USER_UID=10001
 USER ${USER_UID}
 
 COPY --from=prep /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
-COPY otelcontribcol /
+COPY otelcontribcol /otelcol-contrib
```

Comment:
their Dockerfiles are confusing... the helm chart was expecting to run the command `/otelcol-contrib` but the binary that gets built and copied into the image is `/otelcontribcol`?